### PR TITLE
Added info on multiple IPs for egressIPs in case of failover

### PR DESCRIPTION
**Pull request by bfallonf**
I think this is all that's needed for: https://trello.com/c/dRj7BLeb/818-document-ha-for-auto-egress-ip-egress

Will verify over email.

cc @aheslin @danwinship @knobunc

### CODE REVIEW

**Comment by bfallonf** [comment]
Thanks @danwinship . I've changed to your suggestion with some minor rewording.

@bmeng can I ask for a QE review, please? It's a small change, but would like the thumbs up.

File: admin_guide/managing_networking.adoc [hunk 1]
```diff
@@ -1204,8 +1204,12 @@ For example, to assign the `MyProject` project to an IP address of
...
 ----
 +
-The `egressIPs` field is an array, but must be set to a single IP address. If
-setting multiple IPs, the other IPs will be ignored.
+The `egressIPs` field is an array. While in earlier releases it could only
+contain a single IP address, as of {product-title} version 3.10 `egressIPs` can
+be set to two or more IP addresses on different nodes to provide high
+availability. If multiple egress IP addresses are set, pods use the first IP in
+the list for egress, but if the node hosting that IP address fails, pods will
+switch to using the next IP in the list after a short delay.
```
Comment (gaurav-nelson) [inline admin_guide/managing_networking.adoc]: maybe `s/pods will switch to using the next/pods will use the next or /pods will switch to the next`

**Comment by gaurav-nelson** [comment]
@bfallonf added a comment otherwise LGTM 🍰

**Comment by danwinship** [comment]
lgtm

**Comment by bfallonf** [comment]
Thanks, @gaurav-nelson @danwinship . Looks like this is for a z-stream release on August 28. I will hold off on merging until then.

**Comment by kalexand-rh** [comment]
After chatting with @vikram-redhat, we think this one should have been applied to 3.11, too.

**Pull request merged**